<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerFromCb</class>
 <widget class="QDialog" name="LayerFromCb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create layer from clipboard data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>932</width>
       <height>227</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Field delimiters</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="cb_fdTab">
               <property name="text">
                <string>Tab</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_fdComma">
               <property name="text">
                <string>Comma</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_fdSpace">
               <property name="text">
                <string>Space</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Others</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_separator">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Options</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="cb_firstRecord">
               <property name="text">
                <string>First line has field names</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_decimalSep">
               <property name="text">
                <string>Decimal separator is comma</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>307</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="gbGeom">
          <property name="minimumSize">
           <size>
            <width>739</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Geometry</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>X field</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbFieldx">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y field</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbFieldy">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>38</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QgsProjectionSelectionWidget" name="cb_crs"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>7</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Layer name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ed_layerName">
            <property name="text">
             <string>Layer from clipboard</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btPaste">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Paste from Clipboard</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btCreate">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Create layer</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>32</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cb_decimalSep</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LayerFromCb</receiver>
   <slot>optionsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fdComma</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LayerFromCb</receiver>
   <slot>optionsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fdSpace</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LayerFromCb</receiver>
   <slot>optionsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fdTab</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LayerFromCb</receiver>
   <slot>optionsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_separator</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LayerFromCb</receiver>
   <slot>optionsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_firstRecord</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LayerFromCb</receiver>
   <slot>optionsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>optionsChanged()</slot>
 </slots>
</ui>
